File: Kotlin/src/ExperimentResult.kt
class ExperimentResult(var Problem: String, var mes: LongArray, var avg: Double, var med: Long, var h: Long, var l: Long) {

    fun PrintMe(){
        println(Problem+" results: ")
        println("Median: "+med+"ns")
        println("Average: "+avg+"ns")
        println("High: "+h+"\t\tLow: "+l)
        println("======================\n")
    }


}

File: Scalas/src/main/scala/ExperimentResult.scala
package problem

class ExperimentResult(name: String, measurements: List[Long], avg: Double, med: Long, high: Long, low: Long) {

  def PrintMe(): Unit ={
    println(this.name+ "Results: ")
    println("Average: "+this.avg+"ns")
    println("Median: "+this.med+"ns")
    println("High: "+this.high+"ns")
    println("Low: "+this.high+"ns")
    println("")
    println("====================")
    println("")
  }

  def GetName(): String ={ name }

  def GetAvg(): Double={ avg }

  def GetMed(): Long = { med }

  def GetHigh(): Long = { high }

  def GetLow(): Long = { low }

}
